test.css

```diff
@@ -0,0 +1,127 @@
+body, html {
+  margin: 0;
+  padding: 0;
+  height: 100%;
+  background-color: black;
+}
+
+.background-layer {
+  position: absolute;
+  top: -110px;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-image: url('./hackerWaterfall.png');
+  opacity: 0.2;
+  z-index: 0; /* Ensure the background layer is behind content */
```

Comment:
Instead of setting z-index to 0 (and then setting everything everywhere else to 1) you can set it here to -1. This is only ok since this is just a background. This helps you avoid setting every other element's z-index to 1 (note: the default value for z-index is 0).